### PR TITLE
fixed cat widget

### PR DESCRIPTION
**Pull request by javisantana**
@xavijam this is the bug when creating category widgets. I didn't change the spec

### CODE REVIEW

**Comment by xavijam** [comment]
cc @viddo @alonsogarciapablo

**Comment by xavijam** [comment]
cc @viddo @alonsogarciapablo @acanimal

File: src/widgets-service.js [hunk 1]
```diff
@@ -36,7 +36,8 @@ WidgetsService.prototype.newCategoryModel = function (attrs, layer) {
   var dataviewModel = this._dataviews.createCategoryModel(layer, {
     type: 'category',
     column: attrs.column,
```
Comment (acanimal) [inline src/widgets-service.js]: This is a good place to ask: what is the difference between `column` and `aggregationColumn` ?

Comment (alonsogarciapablo) [inline src/widgets-service.js]: I think `column` is the name of the column that is used to get the names of the categories, while `aggregationColumn` is the column that is used to aggregate results by.

Comment (xavijam) [inline src/widgets-service.js]: BTW, as @javisantana pointed, we should change that value name to ```aggregation_column```, just to be consistent if later is it exposed in the viz.json.

Comment (viddo) [inline src/widgets-service.js]: Yes, but let's attack and resolve that separately :point_right: #96

**Comment by xavijam** [comment]
Just using default ```column``` value if ```aggregationColumn``` is not provided.
Happy to merge?

**Comment by viddo** [comment]
:shipit:

**Comment by xavijam** [comment]
:face_with_rolling_eyes:

**Pull request merged**